app/controllers/admin_controller.rb

```diff
@@ -33,8 +33,9 @@ def articles
   end
 
   def update_order
-    object = params[:object].titleize.constantize
-    @obj = object.find(params[:obj_id])
+    # Safely identify the model we're updating the position of
+    klass = [Category, Doc].detect { |c| c.name.casecmp(params[:object]) == 0 }
```

Comment:
Wanted to draw your attention to this and confirm that these are the only two models for which we can "update_order".